src/yunohost/tools.py

```diff
@@ -162,6 +162,28 @@ def tools_maindomain(auth, new_domain=None):
         logger.warning("%s" % e, exc_info=1)
         raise MoulinetteError(errno.EPERM, m18n.n('maindomain_change_failed'))
 
+    # Set hostname
+    pretty_hostname = "(YunoHost/%s)" % new_domain
```

Comment:
No particular reason, it's just to have a "pretty" hostname, like not just the FQDN alone but something describing the hostname and machine, if I understand correctly